MonoGame.Framework/Content/ContentExtensions.cs

```diff
@@ -33,8 +33,7 @@ public static PropertyInfo[] GetAllProperties(this Type type)
 #if WINRT
             PropertyInfo[] infos= type.GetTypeInfo().DeclaredProperties.ToArray();
             var nonStaticPropertyInfos = from p in infos
-                                         where (p.GetMethod != null) && (!p.GetMethod.IsStatic) &&
-                                         (p.GetMethod == p.GetMethod.GetRuntimeBaseDefinition())
```

Comment:
It was trying to remove duplicate properties when one classes overrides the property from a base class.

This is used from the `ReflectiveReader/Writer`.  So some additional tests there would help make me feel better about this change.

Another idea is to wrap this change in a `#if WINDOWS_UAP` so that it only affects Win10.